ngonoessono_a10.js

```diff
@@ -0,0 +1,18 @@
+/*Gertrude Vanessa Ngono Essono
+ngonoessono_a10.js
+INFO2124
+Thoendel
+02/21/2020
+*/
+
+
+
+document.getElementById('action').addEventListener('click', function() {
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    document.getElementById("photo").src= "02.jpg"; //this code will change the original image to 02.jpg when use click on the action button//
+
+
+    document.getElementsByTagName('h1')[0].setAttribute("class", "greenText"); //this code will set h1 to greenText//

```

Comment:
good! These are working now for a single change (1 to 2). What would it take to toggle back and forth?